stubs/waitress/waitress/wasyncore.pyi

```diff
@@ -77,20 +78,21 @@ class dispatcher_with_send(dispatcher):
 
 def close_all(map: Mapping[int, socket] | None = ..., ignore_all: bool = ...) -> None: ...
 
-class file_wrapper:
-    fd: BytesIO = ...
-    def __init__(self, fd: BytesIO) -> None: ...
-    def __del__(self) -> None: ...
-    def recv(self, *args: Any) -> bytes: ...
-    def send(self, *args: Any) -> bytes: ...
-    def getsockopt(self, level: int, optname: int, buflen: bool | None = ...) -> int: ...
-    read: Callable[..., bytes] = ...
-    write: Callable[..., bytes] = ...
-    def close(self) -> None: ...
-    def fileno(self) -> BytesIO: ...
+if sys.platform == "linux" or sys.platform == "darwin":
```

Comment:
```suggestion
if sys.platform != "win32":
```
Consistent with the other check you added.

Reply:
When the original code does `if os.name == "posix"`, the stub did `if sys.platform == "linux" or sys.platform == "darwin"` (there's another platform check like this in `waitress.trigger`).

I added `if sys.platform != "win32"` to be consistent with other `if hasattr(socket, "AF_UNIX")` checks.

But I can change them all to be either if you'd like.

Reply:
I think the runtime checks are pretty much equivalent, and in stubs we have tended to assume all non-Windows OSes are Unix.

Reply:
Sounds good, I'll make all 3 `!= "win32`